pkg/recommender.py

```diff
@@ -425,3 +436,68 @@ def record_data(self, data):
 
     storing_db.close()
 
+  def _schedule_evt(self):
+    '''
+    Send the morning message at 10 am
+    '''
+
+    while True:
+      t = datetime.now()
+      if (t.hour == 10 and t.minute == 2): #CHANGE TO SPECIFIC HOUR WE WANT TO SEND MESSAGE
+        # sending action to phone
+        try:
+          # time sending the message
+          time1 = str(int(time.time()))
+          time_sent = str(datetime.fromtimestamp(int(time1)))
+
+          # items needed in url
+          pre_empathid = '999|' + time1
+
+          phone_url = 'http://191.168.0.106:2226'
+          server_url = 'http://191.168.0.107/ema/ema.php'
+          androidid = 'db7d3cdb88e1a62a'
+          alarm = 'true'
+
+          url_dict = {
+            'id': '1',#CHANGE THIS LATER
```

Comment:
Since this message is not dispatched by Ashley, there is no speaker id. I am defaulting the speaker id to 1. Is that okay?